gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/DiskStoreImpl.java

```diff
@@ -4811,16 +4814,16 @@ public void run() {
           SystemFailure.checkFailure();
           indexRecoveryFailure.compareAndSet(null, err);
         } finally {
-          for (Oplog oplog : this.allOplogs) {
-            oplog.clearInitRecoveryMap();
-          }
+//          for (Oplog oplog : this.allOplogs) {
+//            oplog.clearInitRecoveryMap();
+//          }
           markIndexRecoveryDone();
```

Comment:
Doing it later for rebuilding. But yes after recent changes I think I cannot remove it. Will see. Anyways another pull request created for 9index sanity checks related changes. snap-588